erp.py

```diff
@@ -64,11 +72,14 @@ def wrapped_func(*args, **kwargs):
             'requestedUrl': 'https://erp.iitkgp.ernet.in/IIT_ERP3/welcome.jsp',
         }
 
-
         r = s.post(ERP_LOGIN_URL, data=login_details,
                    **req_args)
+
+        if len(r.history) < 2:
+            raise SecretAnswerError
```

Comment:
It would be great if this error would inform the user about the question and the incorrect answer so that the user can directly fix the problem and run again.

Reply:
Done!